diff --git a/image/softwarefactory.install b/image/softwarefactory.install
index 831ec12..3da5d2e 100755
--- a/image/softwarefactory.install
+++ b/image/softwarefactory.install
@@ -32,7 +32,7 @@ if test -f /var/lib/sf/zuul-rpm-build/repodata/repomd.xml; then
 fi
 
 if [ "${USE_SWIFT_CACHE}" == "yes" ]; then
-    ENABLED_REPOS+="sfbase,sfextras,sfupdates,sfepel,sfrdo,sfmaster"
+    ENABLED_REPOS+="sfbase,sfextras,sfupdates,sfrdo,sfmaster"
     DISABLED_REPOS="*"
     INSTALL_OPTIONS="--disablerepo=${DISABLED_REPOS} --enablerepo=${ENABLED_REPOS}"
     ANSIBLE_EXTRA_VARS="yum_disable_repo=* yum_enable_repo=${ENABLED_REPOS}"
@@ -105,7 +105,7 @@ EOF
     if [ ! -f "${dir}/usr/bin/yum" ]; then
         rpm --root="${dir}" --import $(ls ${dir}/etc/pki/rpm-gpg/*.key | fgrep -v Debug)
         # We cannot use install_packages since the basesystem isn't yet setup
-        yum --disableplugin=* --installroot ${dir} ${INSTALL_OPTIONS} install -y basesystem yum shadow-utils yum-plugin-priorities
+        yum --disableplugin=* --installroot ${dir} ${INSTALL_OPTIONS} install -y basesystem yum shadow-utils
         echo "/dev/root / ext4 errors=remount-ro 0 1" > ${dir}/etc/fstab
         checkpoint "first bootstrap"
     fi
@@ -148,27 +148,25 @@ enable_selinux() {
 
 install_sf_packages() {
     cat > ${dir}/etc/yum.repos.d/sf-repos.repo <<EOF
-[sfepel]
-name=Extra Packages for Enterprise Linux 7
-baseurl=http://46.231.132.68:8080/v1/AUTH_b50e80d3969f441a8b7b1fe831003e0a/repomirror/epel/
+[sfops]
+name=openstack-newton
+baseurl=http://46.231.132.68:8080/v1/AUTH_b50e80d3969f441a8b7b1fe831003e0a/repomirror/opstools/
 gpgcheck=1
 enabled=0
-priority=105
-exclude=python2-jenkins-job-builder
 
 [sfrdo]
 name=openstack-newton
 baseurl=http://46.231.132.68:8080/v1/AUTH_b50e80d3969f441a8b7b1fe831003e0a/repomirror/openstack-newton/
 gpgcheck=1
 enabled=0
-exclude=python2-jenkins-job-builder
 
 [sfmaster]
 name=SF master packages
-baseurl=http://46.231.133.231/kojifiles/repos/${SF_REPO}/
+# TODO: uncomment this (DO NOT MERGE)
+# baseurl=http://46.231.133.231/kojifiles/repos/${SF_REPO}/
+baseurl=http://46.231.133.231/kojifiles/repos/sf-master-centos7/
 gpgcheck=0
 enabled=1
-priority=110
 EOF
     # Check for testing packages
     if test -n "${TESTING_URL}"; then
@@ -178,7 +176,6 @@ name=SF testing packages
 baseurl=${TESTING_URL}
 gpgcheck=0
 enabled=0
-priority=2
 EOF
     fi
     # Check for local testing packages
@@ -189,7 +186,6 @@ name=SF local testing packages
 baseurl=file://${LOCAL_TESTING}/
 gpgcheck=0
 enabled=0
-priority=1
 EOF
         mkdir -p "${dir}/${LOCAL_TESTING}"
         rsync -a --delete "${LOCAL_TESTING}/" "${dir}/${LOCAL_TESTING}/"
diff --git a/rpm-requirements.sh b/rpm-requirements.sh
index 7e3f8f8..41c560f 100755
--- a/rpm-requirements.sh
+++ b/rpm-requirements.sh
@@ -19,11 +19,6 @@ which pip &> /dev/null          || PKGS="${PKGS} python-pip"
 [ -f "/usr/lib64/libvirt/connection-driver/libvirt_driver_lxc.so" ] || PKGS="${PKGS} libvirt-daemon-driver-lxc libvirt-daemon-lxc libvirt"
 [ -d "/usr/lib/python2.7/site-packages/mock" ] || PKGS="${PKGS} python2-mock"
 
-if [ ! -f "/etc/yum.repos.d/epel.repo" ]; then
-    echo "(+) Adds epel-release..."
-    sudo yum install -y epel-release
-fi
-
 if [ ! -z "${PKGS}" ]; then
     echo "(+) Installing build requirements..."
     sudo yum install -y $PKGS
